grails-app/views/report/_dataTableDialog.gsp

```diff
@@ -1,26 +1,25 @@
 <table id="dialogDataTable" class="dataTable">
     <thead>
         <tr>
-            <th><g:message code="default.date.label"/></th>
-            <th><g:message code="default.time.label"/></th>
-            <th><g:message code="transaction.transactionType.label"/></th>
-            <th><g:message code="transactionType.transactionCode.label"
-                           default="Transaction Code"/></th>
-            <th><g:message code="default.quantity.label"/></th>
-            <th><g:message code="default.balance.label" default="Balance"/></th>
+            <th><g:message code="default.date.label" default="Date"/></th>
+            <th><g:message code="default.time.label" default="Time"/></th>
+            <th><g:message code="transaction.transactionType.label" default="Type"/></th>
+            <th><g:message code="report.transactionType.transactionCode.label" default="Transaction Code"/></th>
+            <th><g:message code="default.quantity.label" default="Quantity"/></th>
+            <th><g:message code="report.quantityBalance.label" default="Balance"/></th>
         </tr>
     </thead>
     <tbody>
 
     </tbody>
     <tfoot>
     <tr>
-        <th><g:message code="default.date.label"/></th>
-        <th><g:message code="default.time.label"/></th>
-        <th><g:message code="transaction.transactionType.label"/></th>
-        <th><g:message code="transactionType.transactionCode.label" default="Transaction Code"/></th>
-        <th><g:message code="default.quantity.label"/></th>
-        <th><g:message code="default.balance.label" default="Balance"/></th>
+        <th><g:message code="default.date.label" default="Date"/></th>
+        <th><g:message code="default.time.label" default="Time"/></th>
+        <th><g:message code="transaction.transactionType.label" default="Type"/></th>
+        <th><g:message code="report.transactionType.transactionCode.label" default="Transaction Code"/></th>
```

Comment:
Let's include the missing code to message.properties instead of adding a brand new code.
```
transactionType.label=Transaction Type 
transactionType.transactionCode.label=Transaction Code
```

Reply:
the `transactionType.label` is already an existing translation code, here I only added the default message 
```
<th><g:message code="transaction.transactionType.label" default="Type"/></th>
```